TESTING.md

```diff
@@ -42,10 +38,6 @@ We use [Github Actions](https://github.com/medic/cht-core/actions) which runs `g
 2. Extract it anyhere
 3. From your cht-core directory, run `npx allure open <path>/allure-report/`.
 
-### GitHub Actions Protractor Run 
-
-The build process compiles our application. Then installs horticulturalist to run the app. This puts us closer to production. Executes `grunt ci-e2e`. Which then installs and runs chromedriver. Runs the protractor tests against the installed app version. Currently there are 3 jobs that execute in the supported node environments.  
-
 ### WebdriverIO GitHub Actions Run
 
 The main difference now is that `grunt ci-webdriver` is executed now instead of `ci-e2e`. This executes the Webdriver IO tests.
```

Comment:
Wondering how we could reformulate this sentence, considering the removal of the lines above. 

Could we just say something like:
`grunt ci-webdriver` is executed to run the Webdriver IO tests. ?